src/net/peacefulcraft/sco/swordskills/utilities/IStrengthWeakness.java

```diff
@@ -0,0 +1,13 @@
+package net.peacefulcraft.sco.swordskills.utilities;
+
+import java.util.List;
+
+/**
+ * Interface for players and mobs that implement
+ * outgoing and incoming damage modifiers.
+ */
+public interface IStrengthWeakness {
+    List<String> getDamageModifiers();
```

Comment:
Is this meant to be a list of things that have `.set...` things in `IDamage`? Further, what is the source of those modifiers? I am assuming a combination of an Entities' base abilities, Items, and SwordSkills, but wanted to clarify.